src/main/kotlin/org/elm/ide/docs/ElmDocumentationProvider.kt

```diff
@@ -152,6 +152,19 @@ private fun documentationFor(decl: ElmModuleDeclaration): String? = buildString
         append(" ", ids.last().text)
         renderDefinedInQIDs(ids.dropLast(1))
     }
+
+    renderDocContent(decl) { html ->
+        val declarations = ModuleScope(decl.elmFile).getDeclaredValues()
+
+        // Render @docs commands
+        html.replace(Regex("<p>@docs (.+?)</p>")) { match ->
```

Comment:
Elegant solution 💯. I was expecting that doing something like this would require lexing the contents of block comments and creating PsiElements, but this is a great, non-invasive solution for right now.

Eventually I _do_ want to parse it so that we can have references from identifiers in doc comments to identifiers in source code. Right now the rename refactoring is blind to these "soft" linkages.